main.py

```diff
@@ -10,7 +10,7 @@
 import arcrest
 import dateutil.parser
 import dateutil.tz
-from arcrest.manageorg._community import Group
+from arcrest.manageorg._community import Group  # @UnusedImport
```

Comment:
Are these `#@Unused…` directives specific to Eclipse?

Are all of the warnings you see about unused symbols incorrect?

I don't remember seeing these warnings when I worked on this last time.  Is it running under the interpreter with the highest level of warnings enabled?

Reply:
Those are for pylint.  The secrets warning is correct.  I didn't check the group warning since I didn't touch it and didn't want to change things I didn't need to.

If I run with the highest level of warning I get tons of them. For this round I didn't change anything that I didn't touch.